File: support/mock_backend/wiringPi/softServo.c
#include <stdint.h>
#include <stddef.h>
#include <stdbool.h>
#include <string.h>
#include "wiringPi/softServo.h"
void softServoWrite(int pin, int value) { (void)pin;(void)value; }
int softServoSetup(int p0, int p1, int p2, int p3, int p4, int p5, int p6, int p7) { (void)p0;(void)p1;(void)p2;(void)p3;(void)p4;(void)p5;(void)p6;(void)p7;return 1; }

File: tests/wiringPi/softServo.py
import importlib
import pytest

m=importlib.import_module("wiringop.wiring_pi.soft_servo")

def test_soft_servo_write():
    _=m.soft_servo_write(1, 1)

def test_soft_servo_setup():
    _=m.soft_servo_setup(1, 1, 1, 1, 1, 1, 1, 1)
